test/unittests/state_transition_trace_test.cpp

```diff
@@ -16,6 +16,9 @@ TEST_F(state_transition, trace_example)
     expect.status = EVMC_REVERT;
     expect.post[To].exists = true;
 
+    // Example of checking the expected the post state MPT root hash.
```

Comment:
I don't know why you put it into this test, it could be just in a separate test I think.